sentry-spring-boot-starter/src/main/java/io/sentry/spring/autoconfigure/SentryProperties.java

```diff
@@ -0,0 +1,158 @@
+package io.sentry.spring.autoconfigure;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Configuration for Sentry, example:
+ *
+ * <pre>
+ * sentry:
+ *     enabled: true
```

Comment:
Add enabled property to configuration properties, please. I understand, this is not required, but it helps with work with IDE (like IDEA), because IDEA don't understand properties from Conditions, but understand properties from ConfigurationProperties

Reply:
IDEA works well without it, you can check this screenshot:

<img width="642" alt="Xnip2019-10-28_18-34-00" src="https://user-images.githubusercontent.com/139017/67671840-9ffa0100-f9b1-11e9-9f87-5ce2818697a7.png">

You can find this and other properties in the `spring-configuration-metadata.json` file, which is auto-generate during build of this library. IDEA is using this json file as source of true.

Reply:
Oh, yes, sorry. I don't use json and ConfigProperties together :-). This is my mistake

Reply:
But I can add it to the Properties class, in case if someone need to autowire it as a bean and use isEnabled method.

Reply:
It will be great :-)

Reply:
If you will do this, maybe, you add property `sentry.init-default-client` to SentryProperties too? And after that you can remove `.json` file - all properties will be described in `SentryProperties` class

Reply:
Done.